client/src/components/SearchBar/style.module.css

```diff
@@ -0,0 +1,13 @@
+.container {
```

Comment:
Can we use .scss files for styling going forward

Reply:
Sure happy to have a look. Can we get some documentation on this styling method tho? This is not something taught in 750

Reply:
You simply change the `.css` extension to `.scss`. All CSS is valid SCSS so the content does not have to change

Reply:
Easy then. This needs to be documented though in the wiki.